akka-stream/src/main/mima-filters/2.5.x.backwards.excludes

```diff
@@ -210,3 +211,17 @@ ProblemFilters.exclude[IncompatibleMethTypeProblem]("akka.stream.impl.FanoutProc
 ProblemFilters.exclude[IncompatibleMethTypeProblem]("akka.stream.impl.FanoutProcessorImpl.props")
 ProblemFilters.exclude[IncompatibleMethTypeProblem]("akka.stream.impl.GraphStageIsland.this")
 ProblemFilters.exclude[IncompatibleMethTypeProblem]("akka.stream.impl.fusing.GraphInterpreterShell.this")
+# internal and only used in one test
+ProblemFilters.exclude[MissingClassProblem]("akka.stream.NoMaterializer$")
+ProblemFilters.exclude[MissingClassProblem]("akka.stream.NoMaterializer")
```

Comment:
Why can't we move it to tests?

Reply:
Did move it to the test it is used in, we still need to add the mima exclude

Reply:
Oops, sorry, didn't look closely enough.